web/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Comment:
hey what do you think about removing the web folder and moving the front end to a different repository? I think it will be easier for deployment